ots-cli.js

```diff
@@ -126,6 +129,7 @@ const upgradeCommand = program
     }
     options = parseCommon(options)
     options.calendars = options.calendar
+    options = parseCommon(options)
```

Comment:
nit: I think this is not needed

Reply:
Indeed not needed, too much copy-paste :)